tools/code_format/check_format.py

```diff
@@ -159,6 +159,16 @@ def evaluate_lines(self, path, line_xform, write=True):
         format_flag = True
         output_lines = []
         for line_number, line in enumerate(self.read_lines(path)):
+            if line_number == 0 and path.endswith(".h"):
+                if line == "":
+                    error_message = "%s:%d: %s" % (
+                        path, line_number + 1, "the first line is empty for header files")
+                    continue
```

Comment:
Isn't this part redundant (and a weird unhelpful error message that will likely lead to getting the one about `#pragma` afterwards)?

Why not drop this condition and just have the error message on the other one be `"#pragma once must be the first line of header files"`

Reply:
(Realizing it's maybe to do with the autocorrection.)

Reply:
(deleted suggestion after reading actual file)